book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
```

Comment:
- [ ] Good job for adding the method here but I think the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association so kindly pass it here 🙏🏻✨🔥

`Rental.new(date, self, person)`